apps/passport-client/components/modals/SettingsModal.tsx

```diff
@@ -48,13 +48,19 @@ export function SettingsModal({
               {hasSetupPassword ? "Change" : "Add"} Password
             </LinkButton>
             <Spacer h={16} />
+            <AccountExportButton />
+            <Spacer h={16} />
+            <LinkButton $primary={true} to="/import" onClick={close}>
+              Import Account Data
+            </LinkButton>
+            <Spacer h={16} />
```

Comment:
I'm not sure this spacer made a difference as the modal is still the same size for me. I wonder if its height is hard-coded somewhere.